packages/table/src/select-table-header-cell.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
```

Comment:
move all subcomponents to `components` folder, i.e. `packages/table/src/components`